src/runtime/pkg/katautils/tracing.go

```diff
@@ -30,7 +30,7 @@ var _ export.SpanExporter = (*kataSpanExporter)(nil)
 // ExportSpans exports SpanData to Jaeger.
 func (e *kataSpanExporter) ExportSpans(ctx context.Context, spans []*export.SpanData) error {
 	for _, span := range spans {
-		kataUtilsLogger.Infof("Reporting span %+v", span)
+		kataUtilsLogger.Tracef("Reporting span %+v", span)
```

Comment:
@cmaf - from talking with @jodh-intel  and viewing @liubin 's work, I understand this is utilized just for test. Heads up that we'll need to update the test configuration to utilize `Trace`  instead of `Info`, if that is the case.